packages/list.lua

```diff
@@ -0,0 +1,107 @@
+SILE.scratch.listStack = {}
+
+SILE.settings.declare({
+  name = "list.bullet.numbered",
+  type = "string",
+  default = ".",
```

Comment:
So it's a dot. Weird name then, `list.bullet.numbered` (you are numbering bullets as dots? Heh.)